Doberman/hypervisor.py

```diff
@@ -142,8 +142,14 @@ def hypervise(self) -> None:
 
     def send_remote_heartbeat(self) -> None:
```

Comment:
Looks fine, but if we want to make it more generic should probably put the destination folder in the mongo DB as well. And maybe allow multiple remote hosts? (should be easy to change I think, I wouldn't spend a lot of time on it...)

Reply:
Ok then we need to keep in mind to check our pipelines with digital control nodes before the next run.